views/building-hours/all-building-hours.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import {
+  Text,
+} from 'react-native'
+
+import moment from 'moment'
+
+export function allBuildingHours(info, style) {
+  let dayTimes = []
+  let hoursString = ''
```

Comment:
Why are you defining `hoursString` all the way out here? Could you move it a bit closer to where it's used, or at least to inside the loop? Thx.